packages/lxd/test/cluster/delete.coffee

```diff
@@ -0,0 +1,110 @@
+
+nikita = require '@nikitajs/core/lib'
+{tags, config} = require '../test'
+they = require('mocha-they')(config)
+path = require('path')
+
+return unless tags.lxd
+
+describe 'lxc.cluster.delete', ->
+
+  they 'delete a cluster', ({ssh}) ->
+    @timeout -1 # yum install take a lot of time
+    nikita
+      $ssh: ssh
+    , ->
+      cluster =
+        networks:
+          nktlxdpub:
+            'ipv4.address': '192.0.2.1/28'
+            'ipv4.nat': true
+            'ipv6.address': 'none'
+            'dns.domain': 'nikita.local'
+        containers:
+          'nikita-cluster-del-1':
+            image: 'images:centos/7'
+            nic:
+              eth0:
+                name: 'eth0', nictype: 'bridged', parent: 'nktlxdpub'
+          'nikita-cluster-del-2':
+            image: 'images:centos/7'
+            nic:
+              eth0:
+                name: 'eth0', nictype: 'bridged', parent: 'nktlxdpub'
+      await @lxc.cluster cluster
+      await @lxc.cluster.stop {...cluster, wait: true}
+      {$status} = await @lxc.cluster.delete cluster
+      $status.should.be.true()
+      {data} = await @lxc.query
+        path: '/1.0/instances'
+      data.should.not.containEql '/1.0/instances/nikita-cluster-del-1'
+      data.should.not.containEql '/1.0/instances/nikita-cluster-del-2'
+      {data} = await @lxc.query
+        path: '/1.0/networks'
+      data.should.not.containEql '/1.0/instances/nktlxdpub'
+      data.should.not.containEql '/1.0/instances/nktlxdprv'
```

Comment:
same, we could have an `lxc.network.list` action, maybe for another pull request